### DISCUSSION TITLE
Question about braintree vs. paypal dependencies (with the same version)

### DISCUSSION THREAD

**Discussion by erawhctim** [braintree/braintree-android-drop-in#464] [Q&A]
Can someone help confirm if these two artifacts are the same?

- `com.paypal.android.sdk:data-collector:3.21.1`
- `com.braintreepayments.api:data-collector:3.21.1`
- 
We depend on v5.4.2 of the drop-in SDK, which transitively depends on `com.paypal.android.sdk:paypal-one-touch:3.21.1` -> `com.paypal.android.sdk:data-collector:3.21.1`.

Google's Play Store rejections mention `com.braintreepayments.api:data-collector` instead of `com.paypal.android.sdk:data-collector`. We thought that everything would be find as long as we're on v3.21.1, but we're realizing now that the different group names may be what is holding up Google's reviewers internally.

Do we need to do extra work to exclude the `paypal data-collector` dependency and manually add the braintree one? Or are they identical?

**Comment by sshropshire** [comment]
Hey @erawhctim including `com.braintreepayments.api:drop-in:5.4.2` should be enough. The dependencies you mention will be pulled in transitively. I would double check that all tracks (production and test) are updated to the latest build of your app that includes `5.4.2` of DropIn. Also I'd consider filing an appeal.

Google hasn't been very clear with us on this matter, but we have no recent changes to the `3.x` SDK and they assured us that if nothing had changed all should be fine. They are very tight lipped in their communications, and we haven't gotten much explicit guidance from them.

Let us know if you hear more information. We'd like to help resolve this as quickly as we are able.

**Reply by erawhctim** [reply]
Thank you for confirming. Your assumptions match ours. 

I'll let you know if we hear back any more info from our appeal.
